workspaces/github-actions/README.md

```diff
@@ -1,15 +1,5 @@
 # [Backstage](https://backstage.io)
 
-This is your newly scaffolded Backstage App, Good Luck!
```

Comment:
I'd actually probably leave this section here instead of moving further down! These instructions are specifically about how to test the plugin within this local environment / germane to the yarn install & yarn dev instructions below. The readme within the plugin itself should have instructions on how to install the plugin elsewhere imo (similar to how the readmes for any other plugins in this repo that have an example app keep yarn dev instructions in the top level)